test/libraryUtilitiesTest.ts

```diff
@@ -314,6 +314,178 @@ describe('constructLibraryItem function', function () {
     expect(result.childItems[0].childItems.length).to.equal(0);
   });
 
+  /*
+    This test creates a single layoutELement:
+    {
+      ...
+      "include": [ { "path": "a.b" } ],
+      "childElements": []
+    }
+
+    And two typeListNodes:
+    {
+      {
+        "fullyQualifiedName": "a.b",
+        "contextData": "a.b@param1"
+      },
+      {
+        "fullyQualifiedName": "a.b",
+        "contextData": "a.b@param1,param2",
+      }
+    }
+
+    The result should have the following structure:
+    - parent node
+      |- b (overload 1)
+      |- b (overload 2)
+   */
+  it('should construct overloads using a single path', function () {
+    let layoutElement = new LibraryUtilities.LayoutElement(new testClasses.LayoutElementData());
+    layoutElement.include = [];
+    includeInfo1 = { path: 'a.b' };
+    layoutElement.include.push(includeInfo1);
+
+    let typeListNode1 = new LibraryUtilities.TypeListNode(new testClasses.TypeListNodeData());
+    typeListNode1.fullyQualifiedName = 'a.b';
+    typeListNode1.contextData = 'a.b@param1';
+    typeListNodes.push(typeListNode1);
+
+    let typeListNode2 = new LibraryUtilities.TypeListNode(new testClasses.TypeListNodeData());
+    typeListNode2.fullyQualifiedName = 'a.b'; // The overload has the same fullyQualifiedName
+    typeListNode2.contextData = 'a.b@param1,param2';
+    typeListNodes.push(typeListNode2);
+
+    result = LibraryUtilities.constructLibraryItem(typeListNodes, layoutElement);
+    expect(result.childItems.length).to.equal(2);
+    expect(result.childItems[0].text).to.equal('b');
+    expect(result.childItems[0].contextData).to.equal('a.b@param1');
+    expect(result.childItems[1].text).to.equal('b');
+    expect(result.childItems[1].contextData).to.equal('a.b@param1,param2');
+  })
+
+  /*
+    The following test creates a nested layoutElement:
+    {
+      ... //  parent item 1
+      "include": []
+      "childItems": [
+        {
+          ... // parent item 2
+          "include": [ { "path": "a.b.c" } ],
+          "childItems": []
+        }
+      ]
+    }
+
+    And two typeListNodes:
+    {
+      {
+        "fullyQualifiedName": "a.b.c.d"
+      },
+      {
+        "fullyQualifiedName": "a.b.c.e"
+      }
+    }
+
+    The result should have the following structure:
+    - parent item 1 (result)
+      |- parent item 2
+          |- c
+             |- d
+             |- e
+  */
```

Comment:
Do we have another test that has `inclusive = false`? In which case I expect it to be (without `c`):

  - parent item 1
    - parent item 2
      - d
      - e